src/main/java/net/sf/jabref/gui/keyboard/KeyBindingCategory.java

```diff
@@ -23,7 +23,7 @@
 
 
     private KeyBindingCategory(String name) {
-        this.name = name;
+        this.name = name.replace("&", "");
```

Comment:
I know Olly made you change this, but actually your previous fix was better.
The KeyBindingCategory is used as the header in the keybinding dialog and thus has absolutely nothing to do with the menu. That the strings exists in the menu is pure (kind of) coincidence. Thus it should be indeed `FILE(Localization.lang("File"))` in `KeyBindingCategory` instead of using `menuTitle` there.